### PR TITLE
[MI-1707] Updated en.yml with marketplace listing content

### PR DESCRIPTION
**Pull request by kcasas**
/cc @zendesk/mintegrations @zendesk/i18n

### Description
Updated en.yml with marketplace listing content

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-1707

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

**Comment by dadah89** [comment]
@kcasas @samgavinio @jmramos02 those multiline strings are invalid, could you fix them before Friday's string sweep? https://github.com/zendesk/rosetta/pull/1479#discussion_r124679812

Please wait for +1 from someone in @zendesk/i18n (usually @mariasica) before merging string changes, most times it'll prevent those sort of issues :)

File: src/translations/en.yml [hunk 1]
```diff
@@ -5,10 +5,45 @@ packages:
...
+
+
+### Improve customer satisfaction with better conversations
+
+
+    * Solve tickets faster by having access to relevant customer data from Shopify, such as billing, shipping, and order information, on Zendesk Support ticket pages.
+
+    * Easily navigate to additional data in Shopify by simply clicking the Order ID.
+
+    * Add a Zendesk Feedback Tab to any Shopify-powered site so your customers can contact your Support team quickly and easily.
```
Comment (olenavelykanova) [inline src/translations/en.yml]: Please remove Feedback Tab from the value as it does not exist anymore. Thanks!

Comment (olenavelykanova) [inline src/translations/en.yml]: please also add screenshots

Comment (samgavinio) [inline src/translations/en.yml]: Hi @olenavelykanova, we're addressing this here: https://github.com/zendesk/shopify_app/pull/80 Thanks.

**Pull request merged**